### PR TITLE
fix missing pool tokens bug

### PR DESCRIPTION
**Pull request by pingustar**
#413

### CODE REVIEW

File: src/services/observables/tokens.ts [hunk 1]
```diff
@@ -399,59 +407,73 @@ export const poolTokens$ = combineLatest([
       converter: string;
     } | null>(
       partialPoolTokens.map(async (poolToken) => {
-        const pool = pools.find(
+        let pool: APIPool | Pool;
```
Comment (zavelevsky) [inline src/services/observables/tokens.ts:410]: this code, from here till line 426 - should go in a helper function that returns the pool or null

File: src/services/observables/tokens.ts [hunk 2]
```diff
@@ -399,59 +407,73 @@ export const poolTokens$ = combineLatest([
...
+        const percent = new BigNumber(amount).div(
+          shrinkToken(poolToken.totalSupply, pool.decimals)
+        );
+
+        const balances = await multicall(
+          pool.reserves.map((x) =>
+            buildTokenPoolCall(pool.converter_dlt_id, x.address)
+          )
+        );
+        if (balances) {
```
Comment (zavelevsky) [inline src/services/observables/tokens.ts:440]: will be more readable to check !balances and return null

**Review by zavelevsky — CHANGES_REQUESTED** [review]
suggestions

File: src/services/observables/tokens.ts (hunk 1 above)
Comment (pingustar) [reply src/services/observables/tokens.ts]: done

File: src/services/observables/tokens.ts (hunk 2 above)
Comment (pingustar) [reply src/services/observables/tokens.ts]: done

**Review by zavelevsky — APPROVED** [review]
LGTM, thank you, but consider creating a tokenAddressToToken map and save lots of iterations

**Review by zavelevsky — APPROVED** [review]
Yay!!

**Pull request merged**